package.json

```diff
@@ -21,14 +21,14 @@
     "node": "^10.0.0"
   },
   "dependencies": {
-    "@google-cloud/logging": "^5.1.3",
-    "@google-cloud/storage": "^3.0.2",
+    "@google-cloud/logging": "5.5.5",
+    "@google-cloud/storage": "3.5.0",
     "body": "^5.1.0",
     "express": "^4.17.1",
     "http-status-codes": "^1.3.2",
     "lodash.debounce": "^4.0.8",
     "request": "^2.88.0",
-    "safe-decode-uri-component": "1.2.1-native",
+    "safe-decode-uri-component": "^1.2.1",
```

Comment:
Please don't change the dependency. The `-native` is specifically used to increase perf.

Reply:
Reverted